test/ping/meson.build

```diff
@@ -93,3 +93,6 @@ foreach dst : [ '127.0.0.1' ] + ipv6_dst
 	test(name, cmd, args : args, should_fail : not run_as_root)
   endforeach
 endforeach
+
+jsontest = find_program('test_json.py', 'test/ping/test_json.py', 'test/ping/test_json.py', required : true)
```

Comment:
nit: this could be just 
```python
jsontest = find_program('test_json.py', 'test/ping/test_json.py', required : true)
```
or even
```python
jsontest = find_program('test_json.py', required : true)
```